Resources/Prototypes/_NF/Research/industrial.yml

```diff
@@ -0,0 +1,33 @@
+﻿# Tier 1
+
+# Tier 2
+
+- type: technology
+  id: RapidConstruction
+  name: research-technology-rapid-construction
+  icon:
+    sprite: Objects/Tools/rcd.rsi
+    state: icon
+  discipline: Industrial
+  tier: 2
+  cost: 10000
+  recipeUnlocks:
+  - RCD
+  - RCDAmmo
+  - BorgModuleRCD
```

Comment:
Put it as tier 3, and 15k points.

Reply:
A bit much, don't you think? I think it'll be better as default RCD in advanced tools research in T2 and rechargeable RCD in T3 under this one.

Not to mention you can buy one of atmos ships, it comes with RCD on it and you can get both RCD and one cartridge for it for 20k-ish, if you just buy-sell ship.

Reply:
Definitely can do.

Though why exactly increase it to tier 3? It was pretty heavily under utilized before even as a tier 2 with less price.

Reply:
"rechargeable RCD" you mean the admin tool that is infinite, hell no.

Also if you can name the ship we will go and remove it as no ship should have RCD mapped on it

Ill leave the increase it to tier 3 open a bit for talks if other maintainers want to comment about what they think about it @MagnusCrowe @GreaseMonk @Cheackraze

Reply:
> Also if you can name the ship we will go and remove it as no ship should have RCD mapped on it

https://github.com/new-frontiers-14/frontier-station-14/pull/1106

It's already been removed, but at the Ship Maker's discretion not out of any current know limitation

Reply:
As if cartridge printing would be difficult at tier 3, not to mention you get bluespace bags there so hauling bunch of them won't be problematic. Not to mention that currently RCD only needed for asteroid steam traps. It's not a big deal, no?

Condor. Atmos\SCI vessel. It comes with RCD, one cartridge & axe on it. you can buy it for ~70k and sell ship for ~50k. @dvir001 

If only wizden kept axe tile prying as is it was...

Reply:
Condor will be getting a fix,
Bluespace bags will get some kind of a fix soon I hope to make them more rare / harder to make but thats another PR issue.

RCD Is used for all floor removal as per axe tile changes, we cannot just revert the axe thing without fucking with another system.

Reply:
Asteroids do need a tweak, but thats need to be on another PR